cmd/tikv-ctl/src/main.rs

```diff
@@ -1118,6 +1101,42 @@ fn main() {
                 .takes_value(true)
                 .help("Set the private key path"),
         )
+        // Arguments for local mode.
+        .arg(
+            Arg::with_name("config")
+                .long("config")
+                .takes_value(true)
+                .help("TiKV config path, by default it's <deploy-dir>/conf/tikv.toml"),
+        )
+        .arg(
+            Arg::with_name("data_dir")
+                .long("data-dir")
+                .takes_value(true)
+                .help("TiKV data-dir, check <deploy-dir>/scripts/run.sh to get it"),
+        )
+        .arg(
+            Arg::with_name("skip-paranoid-checks")
+                .required(false)
+                .long("skip-paranoid-checks")
+                .takes_value(false)
+                .help("Skip paranoid checks when open rocksdb"),
+        )
+        // Deprecated arguments.
```

Comment:
Is this just to remind users with old habits?

Reply:
Yes.